cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -252,10 +253,27 @@ instance HasSeverityAnnotation (TraceSendRecv a) where
 instance HasPrivacyAnnotation a => HasPrivacyAnnotation (TraceLabelPeer peer a)
 instance HasSeverityAnnotation a => HasSeverityAnnotation (TraceLabelPeer peer a)
 
+getFetchDecisionSeverityAnnotation :: FetchDecision a -> Severity
+getFetchDecisionSeverityAnnotation fd =
+    case fd of
+      Left FetchDeclineChainNotPlausible     -> Info
+      Left FetchDeclineChainNoIntersection   -> Notice
+      Left FetchDeclineAlreadyFetched        -> Debug
+      Left FetchDeclineInFlightThisPeer      -> Notice
+      Left FetchDeclineInFlightOtherPeer     -> Debug
+      Left FetchDeclinePeerShutdown          -> Notice
+      Left FetchDeclinePeerSlow              -> Notice
```

Comment:
```suggestion
      Left FetchDeclinePeerShutdown          -> Info
      Left FetchDeclinePeerSlow              -> Info
```
These are all not-unusual behaviours of peers. I don't think we want to spam our logs due to what our peers are doing, unless we're trying to debug things.